docs/tutorials/01_electronic_structure.ipynb

```diff
@@ -108,159 +111,478 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### More attributes of the `ElectronicStructureProblem`\n",
+    "\n",
+    "Below we list some additional attributes of our `problem` instance:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
    "metadata": {},
    "outputs": [
     {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "-0.8105479805373279 * IIII\n",
-      "+ 0.1721839326191554 * IIIZ\n",
-      "- 0.22575349222402372 * IIZI\n",
-      "+ 0.17218393261915543 * IZII\n",
-      "- 0.2257534922240237 * ZIII\n",
-      "+ 0.12091263261776627 * IIZZ\n",
-      "+ 0.16892753870087907 * IZIZ\n",
-      "+ 0.045232799946057826 * YYYY\n",
-      "+ 0.045232799946057826 * XXYY\n",
-      "+ 0.045232799946057826 * YYXX\n",
-      "+ 0.045232799946057826 * XXXX\n",
-      "+ 0.1661454325638241 * ZIIZ\n",
-      "+ 0.1661454325638241 * IZZI\n",
-      "+ 0.17464343068300453 * ZIZI\n",
-      "+ 0.12091263261776627 * ZZII\n"
-     ]
+     "data": {
+      "text/plain": [
+       "MoleculeInfo(symbols=['H', 'H'], coords=[(0.0, 0.0, 0.0), (0.0, 0.0, 1.3889487015553204)], multiplicity=1, charge=0, units=<DistanceUnit.BOHR: 'Bohr'>, masses=[1, 1])"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "problem.molecule"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "-1.1169989967540035"
+      ]
+     },
+     "execution_count": 7,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "problem.reference_energy"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "(1, 1)"
+      ]
+     },
+     "execution_count": 8,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "problem.num_particles"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "2"
+      ]
+     },
+     "execution_count": 9,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "problem.num_spatial_orbitals"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "<ElectronicBasis.MO: 'molecular'>"
+      ]
+     },
+     "execution_count": 10,
+     "metadata": {},
+     "output_type": "execute_result"
     }
    ],
    "source": [
-    "qubit_converter = QubitConverter(mapper=JordanWignerMapper())\n",
-    "qubit_op = qubit_converter.convert(main_op)\n",
-    "print(qubit_op)"
+    "problem.basis"
```

Comment:
A lot of these are not outputting anything that can be seen in the notebook (at least when I view it here)

Reply:
Again, the instance strings... I'll leave the discussion to a single thread.

But for the basis in particular this is actually meaningful:
> <ElectronicBasis.MO: 'molecular'>